tests/bson/bson-packedarray-array-access_error-001.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+MongoDB\BSON\PackedArray array access does not allow writing
```

Comment:
This also includes the test for validating the offset type. Considering splitting that into a separate file.

This overlaps with my previous comment about testing errors for undefined offsets (with the correct type).

Reply:
Split up into several tests.